workspaces/adoption-insights/plugins/analytics-module-adoption-insights/README.md

```diff
@@ -0,0 +1,13 @@
+# analytics-module-adoption-insights
+
+Welcome to the analytics-module-adoption-insights plugin!
+
+This plugin hooks into the backstage analytics API and emits the events to Adoption Insights backend.
+
+## Getting started
```

Comment:
Since this PR has analytics-module implementation in-place, maybe can we add some documentation around installation and configuration for this module?

Reply:
Done.